.changeset/silver-horses-check.md

```diff
@@ -0,0 +1,6 @@
+---
+'create-svelte': patch
+'@sveltejs/kit': patch
+---
+
+Extend user tsconfig from generated .svelte-kit/tsconfig.json
```

Comment:
I haven't looked closely, but my assumption is that this is probably not a strictly breaking change, but we would want people to update their apps. Maybe we should mark it as `[breaking]` to make sure people update their apps?

Reply:
they'll see a warning like this:

<img width="511" alt="image" src="https://user-images.githubusercontent.com/1162160/155737711-ab7bf923-ab88-4abd-b05e-1fb063af2aa8.png">

i reckon that'll be more effective than relying on people to read the changelogs